client/src/featureform/register.py

```diff
@@ -772,6 +773,7 @@ def to_source(self) -> SourceVariant:
             description=self.description,
             tags=self.tags,
             properties=self.properties,
+            # TODO: Add job task id stuff?
```

Comment:
Leave out for now. Dont think theres much use for it since its an internal linking